### DISCUSSION TITLE
Problems running automated install of CORE emulator.

### DISCUSSION THREAD

**Discussion by happyuk** [coreemu/core#697] [General]
I've installed Oracle VM VirtualBox v6.1.36 and set it up to run Ubuntu 22.04 (using a plain vanilla installation)

On following the automated install instructions, I've reached a dead end on running the command  'inv install'

https://coreemu.github.io/core/install.html#automated-install

Specifically the bit where it attempts to install grpcio-tools, this is where it consistently fails.

Sorry I have not been able to yet copy the output from the terminal, I am working on this.

Are there any gotchas I need to be aware of?

**Comment by bharnden** [comment]
I would use 20 or 21, there was an issue related to dependencies. I cannot recall offhand, but going back should solve for now.

**Reply by happyuk** [reply]
Version 20.04 of Ubuntu was observed to work as instructed.  Thanks!

For reference here is the link to the iso file:

https://releases.ubuntu.com/20.04.4/ubuntu-20.04.4-desktop-amd64.iso
